.github/merge-fee-bot.yaml

```diff
@@ -6,6 +6,9 @@ payments:
   - amount: 1
     token: BNB
     address: bnb1tqq9llyr3dyjd559dha6z5r5etk3qfwk07m098
+payment
+  # The tolerance for slightly lower amount, in percentage.  Valid range: 95 -- 100
+  amountTolerancePct: 98
```

Comment:
```suggestion
  amountTolerancePct: 95
```

Reply:
I'm fine with that, but in practice tx fee is much less than 1%.